#ifndef LOGINDIALOG_H
#define LOGINDIALOG_H

#include <QDialog>

namespace Ui {
class loginDialog;
}

class loginDialog : public QDialog
{
    Q_OBJECT

public:
    explicit loginDialog(char* Key,QWidget *parent = nullptr);
    ~loginDialog();

private:
    Ui::loginDialog *ui;
};

#endif // LOGINDIALOG_H
